code/controllers/subsystem/communications.dm

```diff
@@ -158,25 +126,28 @@ var/list/radiochannels = list(
 	"JTAC"			= JTAC_FREQ,
 	"Tactics" 		= TACTICS_FREQ,
 
-	SQUAD_NAME_1	= ALPHA_FREQ,
-	SQUAD_NAME_2	= BRAVO_FREQ,
-	SQUAD_NAME_3	= CHARLIE_FREQ,
-	SQUAD_NAME_4	= DELTA_FREQ,
-	SQUAD_NAME_5	= ECHO_FREQ,
+	SQUAD_MARINE_1	= ALPHA_FREQ,
+	SQUAD_MARINE_2	= BRAVO_FREQ,
+	SQUAD_MARINE_3	= CHARLIE_FREQ,
+	SQUAD_MARINE_4	= DELTA_FREQ,
+	SQUAD_MARINE_5	= ECHO_FREQ,
+	SQUAD_MARINE_CRYO		= CRYO_FREQ,
+	SQUAD_MARSOC	= MARSOC_FREQ,
+
 	"Alamo"			= DS1_FREQ,
 	"Normandy"		= DS2_FREQ,
 
 	"Colony"		= COLONY_FREQ
 )
 
 // central command channels, i.e deathsquid & response teams
-#define CENT_FREQS list(ERT_FREQ, DTH_FREQ, PMC_FREQ, DUT_FREQ, YAUT_FREQ, HC_FREQ)
+#define CENT_FREQS list(ERT_FREQ, DTH_FREQ, PMC_FREQ, DUT_FREQ, YAUT_FREQ)
 
 // Antag channels, i.e. Syndicate
 #define ANTAG_FREQS list()
 
 //Depts - used for colors in headset.dm, as well as deciding what the marine comms tower can listen into
-#define DEPT_FREQS list(COMM_FREQ, MED_FREQ, ENG_FREQ, SEC_FREQ, ERT_FREQ, DTH_FREQ, ALPHA_FREQ, BRAVO_FREQ, CHARLIE_FREQ, DELTA_FREQ, ECHO_FREQ, SUP_FREQ, JTAC_FREQ, TACTICS_FREQ, WY_FREQ)
+#define DEPT_FREQS list(COMM_FREQ, MED_FREQ, ENG_FREQ, SEC_FREQ, ERT_FREQ, DTH_FREQ, COLONY_FREQ, ALPHA_FREQ, BRAVO_FREQ, CHARLIE_FREQ, DELTA_FREQ, ECHO_FREQ, CRYO_FREQ, SUP_FREQ, JTAC_FREQ, TACTICS_FREQ, WY_FREQ, HC_FREQ, MARSOC_FREQ)
```

Comment:
colony freq shoudn't be on here, because this is the marine radio tower, same with HC and marsoc, and also ERT and DTH now that i see that. WY can stay cuz of CL